### PR TITLE
Active menu item state for org profile header

### PR DESCRIPTION
**Pull request by adamjgrant**
@andrewconner @cvializ

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ angular.module('kifi')
     },
     templateUrl: 'orgProfile/orgProfileHeader.tpl.html',
     link: function (scope) {
+      scope.state = $state;
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.js:17]: something about exposing the full `$state` object to the template rubs me the wrong way

Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.js:17]: I had originally tried just pulling out `$state.current.activetab` but it wouldn't dynamically update. It sort of treated it as a string literal.

Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.js:17]: I'll try just assigning `$state.current`

Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.js:17]: That didn't work. Angular docs suggest you can use `$route` to watch the route, but I couldn't get this to work, `unknown provider $route` error. Got any ideas?

Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.js:17]: I could also manually assign it on `on $stateChangeSuccess`...

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileHeader.js:17]: eh I'm looking at docs/SO now and I'm starting to think it's a micro-optimization. ship it

**Pull request merged**